axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/AxonServerQueryBus.java

```diff
@@ -499,6 +540,11 @@ public Registration registerDispatchInterceptor(
                     )
             );
             queryExecutor = executorServiceBuilder.apply(configuration, queryProcessQueue);
+            on(QUERY,
+               (ctx, inbound) -> queryExecutor.execute(new QueryProcessor.QueryProcessingTask(inbound.getQuery())));
```

Comment:
This means that we send a confirmation on each query that we receive. Do we really want this?

Reply:
IMO yes. The reason is that we send `QueryResponse` message when the query is executed. Confirmation should be sent as soon as we received the query so we know that the communication is working.